### ISSUE TITLE
Question about blurred image sets

### ISSUE THREAD

**Issue by sesmae** [sungikchoi/NVB#1]
Thanks for sharing the code for your amazing paper. I went over baseline_main_nobar.py . I have a question about it. it seems that in each call to baseline_main_nobar.py you are initializing three networks. f, g0 and g1. does this mean that b_train = 1 in all of the experiments in the script? Because based on your paper there need to be multiple randomly initialized target networks for each set of blurred images.But it seems that in each experiment only one set of blurred CIFAR20 images are generated by zeroing the bottom Ki non-zero singular values for each image. Am I missing or misunderstood something here?

**Comment by sungikchoi** [comment]
Yup. In the CIFAR-10 dataset, we fixed the number of the blurred datasets (b_train) to 1. However, in LSUN or TinyImageNet dataset, or replication of experiment where we used uniform log effective rank criteria, the number of the blurred datasets becomes larger, which means you need more randomly initialized g's.

**Comment by sesmae** [comment]
> Yup. In the CIFAR-10 dataset, we fixed the number of the blurred datasets (b_train) to 1. However, in LSUN or TinyImageNet dataset, or replication of experiment where we used uniform log effective rank criteria, the number of the blurred datasets becomes larger, which means you need more randomly initialized g's.

Great! Thank you!
